src/sentry_plugins/trello/plugin.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import absolute_import
+
+import re
+from django.conf.urls import url
+from rest_framework.response import Response
+
+
+from sentry.utils.http import absolute_uri
+from sentry.exceptions import PluginError
+from sentry.plugins.bases.issue2 import IssuePlugin2, IssueGroupActionEndpoint
+from sentry_plugins.base import CorePluginMixin
+from .client import TrelloApiClient
+
+
+SETUP_URL = "https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/trello/Trello_Instructions.md"  # NOQA
+
+
+class TrelloPlugin(CorePluginMixin, IssuePlugin2):
+    description = "Create Trello cards on issues"
+    slug = "trello"
+    title = "Trello"
+    conf_title = title
+    conf_key = "trello"
+    auth_provider = None
+    resource_links = [("Trello Setup Instructions", SETUP_URL)] + CorePluginMixin.resource_links
+
+    def get_config(self, project, **kwargs):
+        """
+        Return the configuration of the plugin.
+        Pull the value out of our the arguments to this function or from the DB
+        """
+
+        def get_value(field):
+            initial_values = kwargs.get("initial") or {}
+            if initial_values.get(field):
```

Comment:
This line and the remaining can be combined:

```suggestion
            return initial_values.get(field, self.get_option(field, project))
```